templates/terraform/resource.html.markdown.erb

```diff
@@ -80,6 +80,7 @@ To get more information about <%= object.name -%>, see:
 ~> **Warning:** <%= object.docs.warning -%>
 <%- end -%>
 
+<%#- We over-generate examples/oics buttons here; they'll all be _valid_ just not necessarily for this provider version -%>
```

Comment:
I'm not sure what to take away from this. Does the end result have buttons that point to the wrong version provider?

Reply:
We'll generate buttons and examples that are correct & working, but we'll generate `google-beta` examples into the `google` docs. This isn't a problem while the docs are merged, but could be a problem if we split them in the future.

Reply:
makes sense, can you add that to the comment?